src/PresentationalComponents/AppIntro/IntroBody.js

```diff
@@ -61,6 +65,7 @@ const IntroBody = ({ isPreProduction }) => {
           {intl.formatMessage(messages.introTakeTour)}
```

Comment:
We dont even have an obj defined for this, im not sure this component is being used in Advisor